package.json

```diff
@@ -13,18 +13,26 @@
   },
   "dependencies": {
     "classnames": "^2.2.3",
-    "object-assign": "^4.1.0"
+    "object-assign": "^4.1.0",
+    "react": "^15.3.0",
```

Comment:
`react` + `react-dom` should actually be devDependencies. NPM is sorta to blame for this. The problem that can arise is that if when say, React 16 is released, a user installs this module, NPM will then install both React 16 & 15 and will add both to the bundle.js. If you leave them off the dependencies then react-spinkit will just require the react that's already being used. Moving these to the devDependencies is still needed for the examples.